lib/process_settings/testing/helpers.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'process_settings/monitor'
+require 'process_settings/settings'
+require 'process_settings/target_and_settings'
+
+require 'process_settings/testing/monitor'
+
+module ProcessSettings
+  module Testing
+    module Helpers
+      extend self
```

Comment:
I hope we don't need `extend self` here. Do you actually need to include this and call it globally?

Reply:
`extend self` just makes everything available 2 ways. I think we can pick one.
> you include them into the RSpec configuration
But you could equally `extend` them in the RSpec configuration, right?
Maybe I'm missing something. If so, let's hop on a call!

Reply:
I'm not sure if the config has an `extend` the configure block for RSpec looks like the following:

```
require 'some_helper_module'

RSpec.configure do |config|
  config.include SomeHelperModule
end
```

Reply:
Hmm, I'm surprised the `extend self` is doing anything. I'll mess with it on my branch.